README.md

```diff
@@ -4,15 +4,46 @@ A small utility to generate beautiful code images using [the awesome _carbon_ se
 
 ## Install
 
-You can get it from PyPI:
+### Install with pip
+Install from PyPI with pip.
+When using playwright, you must also download the browser binary. This package uses the chromium browser.
 
-```
+```shell
 pip install pybites-carbon
+
+playwright install
+# or only install chromium driver
+playwright install chromium
+```
+
+### Install using uv package manager
+
+```shell
+uv add pybites-carbon
+# or add as a dev dependency
+uv add --dev pybites-carbon
+
+
+uv run playwright install
+# or only install chromium driver
+uv run playwright install chromium
+```
+
+### Install as a uv tool
```

Comment:
I would say "as a standalone tool"

I think this is the more common scenario, so could you put this first?